applications/data_analysis/general_applied_math.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>I worry a little bit about the effort required to create and maintain a resource like this for all basic functionality.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Do you think the depth for the NCL entry is enough? I can remove this if it is going to make the scale too big

Reply:
Maybe leave it in for now and let's see what others think about the balance here.  @anissa111 @jukent 

I'm also not opposed to merging this with the general applications entry and feeling it out from there just wanted to raise it as a potential longer term concern for folks to think about.

Thanks for really getting things rolling with content - this is great!

Reply:
<p>If I had noticed before you made this content, I would have maybe discouraged making a python-first resource for this in geocat-applications, but since it's already made, I'm also not opposed to merging it in.</p>

Reply:
I'll resolve this for now, but let's keep this in mind for future design discussions.